### PR TITLE
Added Greenway Gardens to Catalog

### PR DESCRIPTION
**Pull request by Woliver01**
@mcohenui please review before I continue with the catalog update.

### CODE REVIEW

**Comment by mcohenui** [comment]
@Woliver01 Hey Will. So the program is not running through because there is no New_projects_issue_119_mar_address.csv file in the folder. Did you geocode the original file in the MAR geocoder?

**Comment by Woliver01** [comment]
I did geocode the original file in the MAR geocoder. I had to do it twice though because it didn't produce the file the first time.

**Comment by Woliver01** [comment]
@mcohenui hey Mychal, where did we leave off with this?

**Comment by mcohenui** [comment]
@Woliver01 So my original response still stands, I believe. The program isn't running because it can't find the file. So either there is a misspelling in the macro call or one of the required .csv's is missing. Or both. Try running it again and tracking down the issue using the error statements.

**Comment by Woliver01** [comment]
@ptatian please review.

**Comment by ptatian** [comment]
Thanks, @Woliver01. I will review.

File: Prog/AddNew/Templates/New_projects_issue_119.sas [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**************************************************************************
+ Program:  New_projects_issue_119.sas
```
Comment (ptatian) [inline Prog/AddNew/Templates/New_projects_issue_119.sas]: Should move this program from the **Prog/AddNew/Templates** to **Prog/AddNew**.

File: Prog/AddNew/Templates/New_projects_issue_119.sas [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( MAR, local=n )
+%DCData_lib( RealProp, local=n )
+%DCData_lib( ROD, local=n )
+%DCData_lib( DHCD, local=n )
+
+
+%Add_new_projects(
+  input_file_pre = New_projects_issue_119
```
Comment (ptatian) [inline Prog/AddNew/Templates/New_projects_issue_119.sas]: Add the value `S8-MR` to the Program column in **L:\Libraries\PresCat\Raw\AddNew\New_projects_issue_119_subsidy.csv**.

**Comment by ptatian** [comment]
Thanks, @Woliver01. These changes look good! Please go ahead and finalize this. Follow [the instructions for finalizing](https://urbanorg.app.box.com/notes/295752223069) under **Merge pull request and update programs on L:** and **Final batch submit on SAS1.** Please let me know if you have any questions. cc: @mcohenui

**Pull request merged**